pkg/v1/builder/template/plugintemplates/gomod.tmpl

```diff
@@ -4,283 +4,133 @@ go 1.17
+        github.com/pelletier/go-toml v1.9.4 // indirect
+        github.com/phayes/checkstyle v0.0.0-20170904204023-bfd46e6a821d // indirect
+        github.com/pkg/errors v0.9.1 // indirect
+        github.com/pmezard/go-difflib v1.0.0 // indirect
+        github.com/polyfloyd/go-errorlint v0.0.0-20201127212506-19bd8db6546f // indirect
+        github.com/quasilyte/go-ruleguard v0.3.0 // indirect
+        github.com/quasilyte/regex/syntax v0.0.0-20200407221936-30656e2c4a95 // indirect
+        github.com/rivo/uniseg v0.2.0 // indirect
+        github.com/ryancurrah/gomodguard v1.2.0 // indirect
+        github.com/ryanrolds/sqlclosecheck v0.3.0 // indirect
+        github.com/sanposhiho/wastedassign v0.1.3 // indirect
+        github.com/securego/gosec/v2 v2.6.1 // indirect
+        github.com/shazow/go-diff v0.0.0-20160112020656-b6b7b6733b8c // indirect
+        github.com/sirupsen/logrus v1.8.1 // indirect
+        github.com/sonatard/noctx v0.0.1 // indirect
+        github.com/sourcegraph/go-diff v0.6.1 // indirect
+        github.com/spf13/afero v1.6.0 // indirect
+        github.com/spf13/cast v1.4.1 // indirect
+        github.com/spf13/cobra v1.3.0 // indirect
+        github.com/spf13/jwalterweatherman v1.1.0 // indirect
+        github.com/spf13/pflag v1.0.5 // indirect
+        github.com/spf13/viper v1.10.0 // indirect
+        github.com/ssgreg/nlreturn/v2 v2.1.0 // indirect
+        github.com/stretchr/objx v0.1.1 // indirect
+        github.com/stretchr/testify v1.7.1-0.20210427113832-6241f9ab9942 // indirect
+        github.com/subosito/gotenv v1.2.0 // indirect
+        github.com/tdakkota/asciicheck v0.0.0-20200416200610-e657995f937b // indirect
+        github.com/tetafro/godot v1.4.4 // indirect
+        github.com/timakin/bodyclose v0.0.0-20200424151742-cb6215831a94 // indirect
+        github.com/tomarrell/wrapcheck v0.0.0-20201130113247-1683564d9756 // indirect
+        github.com/tommy-muehle/go-mnd/v2 v2.3.1 // indirect
+        github.com/ultraware/funlen v0.0.3 // indirect
+        github.com/ultraware/whitespace v0.0.4 // indirect
+        github.com/uudashr/gocognit v1.0.1 // indirect
+        golang.org/x/mod v0.5.1 // indirect
+        golang.org/x/net v0.0.0-20211005215030-d2e5035098b3 // indirect
+        golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
+        golang.org/x/text v0.3.7 // indirect
+        golang.org/x/tools v0.1.5 // indirect
+        golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+        gopkg.in/ini.v1 v1.66.2 // indirect
+        gopkg.in/yaml.v2 v2.4.0 // indirect
+        gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+        honnef.co/go/tools v0.1.2 // indirect
+        mvdan.cc/gofumpt v0.1.0 // indirect
+        mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed // indirect
+        mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
+        mvdan.cc/unparam v0.0.0-20210104141923-aac4ce9116a7 // indirect
+)
```

Comment:
Missing newline?

Reply:
The `go.mod` file shouldn't have one. I believe `go mod tidy` will just remove it if I add one, so this just matches our existing `go.mod` files.